docs/en_US/Nnictl.md

```diff
@@ -370,6 +371,26 @@ Debug mode will disable version check function in Trialkeeper.
     nnictl experiment list
     ```
 
+* __nnictl experiment delete__
+
+  * Description
+
+    Delete the data generated by nni experiment.
+
+  * Usage
+
+    ```bash
+    nnictl experiment delete [OPTIONS]
+    ```
+  
+  * Options
+
+  |Name, shorthand|Required|Default|Description|
+  |------|------|------ |------|
+  |id|  False| |ID of the experiment|
```

Comment:
'*' means to remove all experiments, which are run from this computer.

Reply:
Fixed, add --all command.